package.json

```diff
@@ -10,7 +10,7 @@
 		"url": "https://github.com/microsoft/vscode-hydrate.git"
 	},
 	"engines": {
-		"vscode": "^1.35.0"
+		"vscode": "^1.31.0"
```

Comment:
Are we intentionally using an older version?

Reply:
@samiyaakhtar yes, the Kubernetes extension is the limiting factor for versions and it uses ^1.31.0